scripts/abi_check.py

```diff
@@ -148,7 +148,8 @@ def _build_shared_libraries(self, git_worktree_path, version):
         my_environment = os.environ.copy()
         my_environment["CFLAGS"] = "-g -Og"
         my_environment["SHARED"] = "1"
-        my_environment["USE_CRYPTO_SUBMODULE"] = "1"
+        if os.path.exists(os.path.join(git_worktree_path, "crypto")):
```

Comment:
In commit message "Only use submodule if present", could you explain what problems?

e.g. mbed-crypto should use error.c from the parent project when being built as a submodule, and if there is no parent project, but the USE_CRYPTO_SUBMODULE is still set, there is no error.c and the build fails.

Reply:
I've reworded the commit message.